recipes/napari-patchcreator/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "napari-patchcreator" %}
+{% set version = "0.1.4" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/napari-patchcreator-{{ version }}.tar.gz
+  sha256: 08fc176d3224430fbe47d2af9664df8b571216402b68e84ca6f30aeedb384ff4
+
+build:
+  noarch: python
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
+  script_env:
+    - QT_QPA_PLATFORM=offscreen
+
+requirements:
+  host:
+    - pip
+    - python >=3.8
+  run:
+    - napari
+    - napari-plugin-engine >=0.1.4
+    - numpy
+    - python >=3.8
+
+test:
+  imports:
+    - napari_patchcreator
+  commands:
+    - pip list
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://github.com/juglab/napari-patchcreator
```

Comment:
This repo is 404-ing

Reply:
I guess, if you're still interested in maintaining the recipe. In my mind that's probably a dead project, but we can certainly package based on PyPI.

Reply:
@tibuch is this project still maintained?

Reply:
I don't know, but @jdeschamps might know more.

Reply:
Thanks for pinging me. I was not aware it had been made private, I will investigate and come back to you asap.

It should be public (since it is on the napari-hub), and we have plan for it in the future.

Reply:
The repo is public again! No idea what happened...